shardingsphere-kernel/shardingsphere-data-pipeline/shardingsphere-data-pipeline-api/src/test/java/org/apache/shardingsphere/data/pipeline/spi/fixture/DialectDDLGeneratorFixTrue.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.shardingsphere.data.pipeline.spi.fixture;
+
+import java.sql.SQLException;
+import javax.sql.DataSource;
+import org.apache.shardingsphere.data.pipeline.spi.ddlgenerator.DialectDDLGenerator;
+
+public class DialectDDLGeneratorFixTrue implements DialectDDLGenerator {
+    
+    private static final String SHOW_CREATE_SQL = "SHOW CREATE TABLE %s";
+    
+    @Override
+    public String generateDDLSQL(final String tableName, final String schemaName, final DataSource dataSource) throws SQLException {
+        StringBuilder sb = new StringBuilder();
+        sb.append(String.format(SHOW_CREATE_SQL, tableName));
+        return sb.toString();
+    }
+    
+    @Override
+    public String getType() {
+        return "MySQL";
```

Comment:
The `MySQL` type is already existed

Reply:
The added unit test is in the ```shardingsphere-data-pipline-api``` module. Since the interface and implementation are not in the same module, the spi exposed by other ```xxxDialectDDLGenerator``` cannot be obtained, so a new ```DataBaseType``` type